osu.Game/Users/Drawables/UpdateableAvatar.cs

```diff
@@ -46,21 +46,21 @@ public new EdgeEffectParameters EdgeEffect
         protected override double LoadDelay => 200;
 
         private readonly bool isInteractive;
-        private readonly bool showUsernameTooltip;
         private readonly bool showGuestOnNull;
+        private readonly bool showUserPanelOnHover;
 
         /// <summary>
         /// Construct a new UpdateableAvatar.
         /// </summary>
         /// <param name="user">The initial user to display.</param>
         /// <param name="isInteractive">If set to true, hover/click sounds will play and clicking the avatar will open the user's profile.</param>
-        /// <param name="showUsernameTooltip">Whether to show the username rather than "view profile" on the tooltip. (note: this only applies if <paramref name="isInteractive"/> is also true)</param>
```

Comment:
The fact that `showUsernameTooltip` (now `showUserPanelOnHover`) depends on `isInteractive` was pretty important informaton imo. Should probably still be there.